app/models/message_thread.rb

```diff
@@ -167,6 +167,10 @@ def unassign_tag(tag)
     message_threads_tags.find_by(tag: tag)&.destroy
   end
 
+  def any_objects_with_requested_signature?
+    messages.any? { |message| message.any_objects_with_requested_signature? }
```

Comment:
Nie je toto tak, ze ked tam je requested tag tak potom ked to podpises tak uz tam taky tag nebude? Cize staci zistit, ze tam nejaky takyto tag je nie?

Reply:
Je, vieme to tak ulahcit